plugins-root/check_icmp.c

```diff
@@ -540,11 +579,34 @@ main(int argc, char **argv)
 				break;
 			case 'V': /* version */
 				print_revision (progname, NP_VERSION);
-				exit (STATE_UNKNOWN);
+				exit (STATE_OK);
```

Comment:
this might be slipped in and is unrelated to this patch. It should stay on unknown according to our guidelines:
`The --help or --version output should also result in Unknown state. `